### ISSUE TITLE
Error message : Please Check the values

### ISSUE THREAD

**Issue by ChuckLoris18** [fvasco/pinpoi#28]
Hello,
Thank you for this software, it is really usefull.
In my company, we have been using it for a couple of years without any issue.

But today, on a new smartphone, I have placed the *ov2 files on a smartphone, and then, when a research is launched, I get the following message : Validation Error, please check the value.

It is strange beacause our data files is still the same for weeks and weeks.
I wonder if the problem comes from the new Android 10 version on Samsung A40.

**Comment by ChuckLoris18** [comment]
![pinpoi](https://user-images.githubusercontent.com/73890641/97989080-4dca9780-1dde-11eb-829a-4f746c684cee.jpg)

**Comment by fvasco** [comment]
Hi, @ChuckLoris18,
your welcome, I'm happy to hear that.

That message was shown because GPS didn't provide any data.
I shall to ask you a preliminary test, you should to check to the GPS authorization to Pinpoi.
Secondary, it is possible that your phone is not able to detect GPS signal in a closed space, you should start Pinpoi in an open space and enable GPS, latitude and longitude should be automatically filled in few minutes.

As a temporary workaround, you can manually fill coordinates or use [LocationShare](https://github.com/pR0Ps/LocationShare) to copy the `geo` URI, you can search it in Pinpoi.

Please send me your feedback

**Comment by ChuckLoris18** [comment]
Hi @fvasco ,
Thank you for your quick feedback !

I have checked the GPS signal and it seems ok.
Actually, I have a samsung A40 with Android 9 and everything works fine.
I have a Samsung A40 with android 10, and it doesn't work.
The 2 phones are side by side, with same GPS signal.

I am affraid android 10 doesn't allow PINPOI to "read" the data on the ov2 file.
(because the problem is not to open any guidance application, the data of the location doesn't even display)

I have checked on both phones and the authorisations are the same (Localization and storage)

**Comment by ChuckLoris18** [comment]
Do you have any recommandation on the location where to place the ov2 files on the phone ? 
Maybe I am facing issue with the file on the phone.
I used to place the ov2 files on my samsung on : Storage\Download

**Comment by fvasco** [comment]
> Actually, I have a samsung A40 with Android 9 and everything works fine.
> I have a Samsung A40 with android 10, and it doesn't work.
> The 2 phones are side by side, with same GPS signal.

Good test @ChuckLoris18!

> I am affraid android 10 doesn't allow PINPOI to "read" the data on the ov2 file.

Can you explain better?
Are you able to import the ov2 file?
In the collection, you can see the POI count at the bottom.

> Do you have any recommandation on the location where to place the ov2 files on the phone ?

No, sorry.

> Maybe I am facing issue with the file on the phone.

I developed a new version to handle file permission better, bot some too-hard rating on Google Play demoralized me.
If this is the way to go, then I can provide you the apk for a quick test.

We can try to understand the issue better using the tutorial

https://fvasco.github.io/pinpoi/

Are the Unesco sites visible? Can you use GPS with Unesco collection?

**Comment by fvasco** [comment]
@ChuckLoris18 news?

**Comment by ChuckLoris18** [comment]
Hello @fvasco !
Yes ! We have found the problem !

Actually, I needed to disable the position, and enable it again...
I do not know why it wasn't working first, but this manoeuver of disabllng/ enabling position has worked on two phones.

The error message was not guiding me on this path. I thought the problem came from the ov2 file.

Anyway, thank you for your help. We are now able to use PINPOI again !
Best regards
![position](https://user-images.githubusercontent.com/73890641/98542093-adb6b780-2290-11eb-8d0d-d5a81376057e.jpg)
